security/cmd/node_agent/na/nodeagent_test.go

```diff
@@ -166,9 +152,14 @@ func TestStartWithArgs(t *testing.T) {
 			t.Errorf("Test case [%s]: sendCSR is called incorrect times: %d VS (expected) %d",
 				id, c.cAClient.Counter, c.sendTimes)
 		}
-		if c.fileContent != nil && !bytes.Equal(fakeFileUtil.WriteContent["cert_file"], c.fileContent) {
-			t.Errorf("Test case [%s]: cert file content incorrect: %s VS (expected) %s",
-				id, fakeFileUtil.WriteContent["cert_file"], c.fileContent)
-		}
+		// TODO(incfly): add test back after we have Read/Load methods on SecretServer.
```

Comment:
This does not look good, please consider to remove from master branch. You can save it to personal branch if necessary.

Reply:
Removed and reword the todo